nautobot/dcim/views.py

```diff
@@ -159,6 +176,18 @@ def get_selected_objects_parents_name(self, selected_objects):
         selected_object = selected_objects.first()
         if selected_object and selected_object.device:
             return selected_object.device.name
+        if selected_object and selected_object.module:
+            return selected_object.module.display
```

Comment:
Is this an issue that we use `name` for the device but `display` for the module? Should we use `display` for the device too like we do below?

Reply:
It would be fine to use display for the device too here. That would help in case there is no name on a device